tgui/packages/tgui/interfaces/NaniteCloudControl.js

```diff
@@ -253,7 +253,7 @@ export const NaniteCloudBackupDetails = (props, context) => {
                             program_id: program.id,
                             rule_id: rule.id,
                           })} />
-                        {rule.display}
+                        {` ${rule.display}`}<br />
                       </Fragment>
```

Comment:
Replace the whole fragment with a Box, and you won't need a br.

Reply:
Done. Looks to work on testing. Thanks!